src/components/DataDuesAction/index.js

```diff
@@ -283,23 +278,22 @@ const DebtForm = ({
           value="true"
           ref={register}
           label="Yes"
-          isInvalid={!!errors[`debts[${debtId}].beingHarrased`]}
+          isInvalid={_.get(errors, `debts[${debtId}].beingHarrased`)}
         />
         <Form.Check
           id={`beingHarrasedYes${debtId}`}
-          isInvalid={!!errors[`debts[${debtId}].beingHarrased`]}
+          isInvalid={_.get(errors, `debts[${debtId}].beingHarrased`)}
```

Comment:
Just a heads up for the future, it is convenient to avoid the usage of `_.get` lodash to avoid breaking static typing flows, as flow-type _(and I think TS too)_ won't be able to infer types from the return value

While here we don't use static type system it can make things harder for a possible future migration or just make things harder for you to "think" when to use it or not.

Instead we can always go with `foo.bar?`

Reply:
Sure, `_.get` makes the code easier to write as we don't have to check for object presence, but I would keep this in mind.

Thanks!

Reply:
Optional chaining operator will do the trick for you. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

_I found my message a bit misleading cause question mark was on the end of the line, so I was trying to pointing to you_ `foo.bar?.variable?`